grammar.js

```diff
@@ -44,7 +44,7 @@ module.exports = grammar({
 
   inline: $ => [
     $._statement,
-    $._top_level_item,
```

Comment:
I think the `_top_level_item` also should be inlined to reduce an internal tree complexity. Of course if it doesn't produces compilation errors, because I'm not sure by looking only on a changed piece of the grammar :shrug:

Reply:
oops, done